components/common/src/test/java/com/hotels/styx/config/schema/SchemaTest.java

```diff
@@ -29,20 +28,9 @@
 
 import static com.fasterxml.jackson.core.JsonParser.Feature.AUTO_CLOSE_SOURCE;
 import static com.fasterxml.jackson.databind.DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES;
-import static com.hotels.styx.config.schema.SchemaDsl.atLeastOne;
-import static com.hotels.styx.config.schema.SchemaDsl.bool;
-import static com.hotels.styx.config.schema.SchemaDsl.field;
-import static com.hotels.styx.config.schema.SchemaDsl.integer;
-import static com.hotels.styx.config.schema.SchemaDsl.list;
-import static com.hotels.styx.config.schema.SchemaDsl.map;
-import static com.hotels.styx.config.schema.SchemaDsl.object;
-import static com.hotels.styx.config.schema.SchemaDsl.opaque;
-import static com.hotels.styx.config.schema.SchemaDsl.optional;
-import static com.hotels.styx.config.schema.SchemaDsl.or;
-import static com.hotels.styx.config.schema.SchemaDsl.schema;
-import static com.hotels.styx.config.schema.SchemaDsl.string;
-import static com.hotels.styx.config.schema.SchemaDsl.union;
+import static com.hotels.styx.config.schema.SchemaDsl.*;
```

Comment:
Put back the original imports. Perhaps worth configuring your editor?

Reply:
Indeed. For some reason my configuration seems to have slipped back to default on this.